frappe/core/doctype/communication/communication.py

```diff
@@ -448,13 +448,16 @@ def update_parent_document_on_communication(doc):
 
 		# if status has a "Replied" option, then update the status for received communication
 		if ('Replied' in options) and doc.sent_or_received=="Received":
-			parent.db_set("status", "Open")
+			parent.status = "Open"
+			parent.flags.ignore_mandatory = True
+			parent.save()
```

Comment:
Why is `parent.save()` required? 
ideally, the parent's controller validation should not be called in this case.

Reply:
In the Issue DocType, when the received communication is created and status is changed back to Open, the Hold Time (time for which the SLA timer had stopped) is added to the response and resolution time left to solve the ticket. This happens on `update_status` called on `validate` in Issue DocType:
https://github.com/frappe/erpnext/pull/21617/files#diff-708a3b7eee1fdfe6023d06655eea8f30R70-R119